queries.sql

```diff
@@ -1,3 +1,36 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions
+FROM all projects.*/
+SELECT  *
+FROM animals
+WHERE trim(name) like '%mon';
 
+SELECT  *
+FROM animals
+WHERE extract('year'
+FROM date_of_birth) BETWEEN 2016 AND 2019;
+
+SELECT  *
+FROM animals
+WHERE neutered = true
+AND escape_attempts < 3;
+
+SELECT  date_of_birth
+FROM animals
+WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT  name,escape_attempts
+FROM animals
+WHERE weight_kg > 10.5;
+
+SELECT  *
+FROM animals
+WHERE neutered = true;
+
+SELECT  *
+FROM animals
+WHERE name != 'Gabumon';
+
+SELECT  *
+FROM animals
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * from animals WHERE name = 'Luna';
```

Comment:
- [ ] Kindly remove the last query because it isn't part of the requirements for this project. 👍

Reply:
okay